src/server/main.pl

```diff
@@ -116,15 +117,24 @@
         p('TerminusDB is still synchronizing backing store')
     ]).
 
+
```

Comment:
This needs to either add a cut after json log format, or use picat syntax with a head constraint.

Reply:
Done! Thanks